nixos/tests/nexus.nix

```diff
@@ -0,0 +1,32 @@
+# verifies:
```

Comment:
I know where you copied this from :D

Reply:
:-D

Reply:
Change it now to more reasonable comments ;-)